src/test-apps/TestWeaveTunnelBR.cpp

```diff
@@ -2310,6 +2310,75 @@ static void TestTunnelRestrictedRoutingOnTunnelOpen(nlTestSuite *inSuite, void *
     gTunAgent.Shutdown();
 }
 
+static void TestTunnelRestrictedRoutingOnStandaloneTunnelOpen(nlTestSuite *inSuite, void *inContext)
```

Comment:
What is it that we are testing here? I'm not exactly seeing the tests that verify that the semantics of the standalone device are enforced, is that embedded elsewhere in the code?

Reply:
nevermind, I see that adding the case statement in line 2868 implements the actual test.